packages/core/src/cloud/utils.ts

```diff
@@ -11,18 +12,34 @@ export function getAppBaseUrl(baseUrl?: string): string {
   return getBaseUrl(baseUrl).replace(/api\./, "");
 }
 
-export async function getClient({
+export function getClient({
   apiKey,
   baseUrl,
-}: ClientParams = {}): Promise<PlatformApiClient> {
+}: ClientParams = {}): typeof LlamaCloud {
   // Get the environment variables or use defaults
   baseUrl = getBaseUrl(baseUrl);
   apiKey = apiKey ?? getEnv("LLAMA_CLOUD_API_KEY");
 
-  const { PlatformApiClient } = await import("@llamaindex/cloud");
+  if (!apiKey) {
```

Comment:
i remember @llamaindex/cloud didn't work with CJS, that's why there's the dynamic import - is this fixed?

Reply:
hmm will check this